pkgs/development/tools/misc/patchelf/default.nix

```diff
@@ -14,6 +14,10 @@ stdenv.mkDerivation rec {
     sha256 = "14npmdxppmh0ci140w8i8cy7zg1pnqg81a1mdsnza711ab7k36k9";
   };
 
+  patches = [
+    ./PR230-fix-false-alarm-for-non-overlapping-sections.patch
```

Comment:
Can you use fetchpatch from the PR?

Reply:
It was tempting, but there is a note at the top of the patchelf nix file that kept me from doing so:

> Note: this package is used for bootstrapping fetchurl, and thus                                                                
> cannot use fetchpatch! All mutable patches (generated by GitHub or                                                             
> cgit) that are needed here should be included directly in Nixpkgs as                                                           
> files.                                                                                                                         

Since using fetchpatch was my first instinct as well, I updated the commit message with a note that hints on this unexpected circumstance.

Reply:
Oh, good you catched that.